commands/command_config.go

```diff
@@ -196,8 +196,8 @@ func NewCmdConfig(ns string, dc doctl.Config, out io.Writer, args []string, init
 			return CheckSandboxStatus()
 		},
 
-		installSandbox: func(dir string, upgrading bool) error {
-			return InstallSandbox(dir, upgrading)
+		installSandbox: func(c *CmdConfig, dir string, upgrading bool) error {
```

Comment:
To support upgrading from legacy, the install command needs access to the governing access token.  It seemed the easiest way was to pass along the CmdConfig.